planner:
  ros__parameters:
    max_v: 3.5
    max_a: 1.6

    waypoints: [ -50.0, 10.0, 15.0,
                -100.0, 10.0, 40.0,
                -300.0, 10.0, 15.0]
    
    stop_index: 0
